Gemfile.lock

```diff
@@ -1,78 +1,78 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.0)
-      actionview (= 6.0.0)
-      activesupport (= 6.0.0)
-      rack (~> 2.0)
+    actionpack (6.0.2.1)
+      actionview (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+      rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.0)
-      activesupport (= 6.0.0)
+    actionview (6.0.2.1)
+      activesupport (= 6.0.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.0)
-      activesupport (= 6.0.0)
-    activerecord (6.0.0)
-      activemodel (= 6.0.0)
-      activesupport (= 6.0.0)
-    activesupport (6.0.0)
+    activemodel (6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activerecord (6.0.2.1)
+      activemodel (= 6.0.2.1)
+      activesupport (= 6.0.2.1)
+    activesupport (6.0.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.1, >= 2.1.8)
-    builder (3.2.3)
+      zeitwerk (~> 2.2)
+    builder (3.2.4)
     concurrent-ruby (1.1.5)
-    crass (1.0.4)
+    crass (1.0.5)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.8.0)
-    i18n (1.6.0)
+    erubi (1.9.0)
```

Comment:
### [erubi](https://github.com/jeremyevans/erubi)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.8.0 → 1.9.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **7** commits. See the full changes on [the compare page](https://github.com/jeremyevans/erubi/compare/1.8.0...1.9.0).

These are the individual commits:
- (3962595) [CI: Add Ruby 2.6 to the matrix](https://github.com/jeremyevans/erubi/commit/396259533b11aeed7207c47504afe6677987ab86)
- (e33a647) [Test on TruffleRuby on Travis](https://github.com/jeremyevans/erubi/commit/e33a64773990959e70f44046dab1f754c38deb54)
- (01ca84b) [Test JRuby 9.2 on Travis](https://github.com/jeremyevans/erubi/commit/01ca84bbd2dd94eab1e7ac8a756ed860d3d611b3)
- (592cabf) [Use minitest-global_expecations in tests to avoid deprecatio…](https://github.com/jeremyevans/erubi/commit/592cabf3ddafeb65758b01317fe6fd4825e8e8a4)
- (9b7fc05) [Try to get Travis passing](https://github.com/jeremyevans/erubi/commit/9b7fc0566179a25c80635f1ced743f87a47f31cf)
- (3db54fc) [Change default :bufvar from 'String.new' to '::String.new' t…](https://github.com/jeremyevans/erubi/commit/3db54fc93d780c892e64d22a675c82919f96354e)
- (3be9947) [Bump version to 1.9.0](https://github.com/jeremyevans/erubi/commit/3be9947899156c45b9036c703998ebb11311e91e)


</details>